parity/configuration.rs

```diff
@@ -978,8 +978,12 @@ impl Configuration {
 		let data_path = replace_home("", &base_path);
 		let is_using_base_path = self.args.arg_base_path.is_some();
 		// If base_path is set and db_path is not we default to base path subdir instead of LOCAL.
-		let base_db_path = if is_using_base_path && self.args.arg_db_path.is_none() {
+		let base_db_path = if is_using_base_path && self.args.arg_db_path.is_none() && self.args.flag_light {
```

Comment:
Maybe this is a little more readable?

```rust
let base_db_path = if is_using_base_path && self.args.arg_db_path.is_none() {
  if self.args.flag_light {
    …
  } else {
    …
  }
} else if self.args.flag_light {
  …
} else {
  …
}

```

Reply:
Thanks for the reviews today. I updated this based on your suggestions.